### ISSUE TITLE
Ability to like comments

### ISSUE THREAD

**Issue by kmsauter** [szdc/tiktok-api#170]
Would it be possible to implement the ability to like comments on a post/has anyone figured out a way to do this?

**Comment by mrklintscher** [comment]
Yes its possible. Take a look here https://github.com/mrklintscher/TikTokPrivateAPI

**Comment by xgorgon** [comment]
you need valid xgorgon for this

i have x-gorgon generation script in python. generates x-gorgon starting with 030xxxxxx

contact: xgorgon@protonmail.com
